website/.gitignore

```diff
@@ -18,3 +18,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+yarn.lock
+package-lock.json
```

Comment:
package-lock.json should not be added to the `website/.gitignore` because when we add or update a new dependency, we have to commit it.